src/test/java/ubc/pavlab/rdp/repositories/UserGeneRepositoryTest.java

```diff
@@ -180,11 +181,9 @@ public void countDistinctGeneByTierIn_whenTierManualMatch_thenCount() {
     }
 
     @Test
-    public void countDistinctGeneByTierIn_whenTierNotMatch_thenDontCount() {
-
-        int count = userGeneRepository.countDistinctGeneByTierIn( tierService.getEnabledTiers() );
-
-        assertThat( count ).isEqualTo( 0 );
+    public void countDistinctGeneByTierIn_whenTierNotMatch_thenRaiseException() {
```

Comment:
Rename the test, passing an empty collection is supported. I also think that there's a corresponding service change that check for emptiness, It does not have to.

Reply:
In fact, it can be completely reverted.